crates/bdk/src/wallet/mod.rs

```diff
@@ -950,10 +950,10 @@ impl Wallet {
     /// [`insert_txout`]: Self::insert_txout
     pub fn calculate_fee_rate(&self, tx: &Transaction) -> Result<FeeRate, CalculateFeeError> {
         self.calculate_fee(tx)
-            .map(|fee| bitcoin::Amount::from_sat(fee) / tx.weight())
+            .map(|fee| Amount::from_sat(fee) / tx.weight())
     }
 
-    /// Compute the `tx`'s sent and received amounts (in satoshis).
+    /// Compute the `tx`'s sent and received amounts (in [`bitcoin::Amount`]).
```

Comment:
What do you think of this?

```suggestion
    /// Compute the `tx`'s sent and received [`Amount`]s.
```